### ISSUE TITLE
doesn't generate image if it have srcset attribute

### ISSUE THREAD

**Issue by sasha8-1** [tsayen/dom-to-image#221]
### Use case: description, code 
 `<img>` tag doesn't work well if it have `srcset` attribute

![peek 2018-06-01 14-40](https://user-images.githubusercontent.com/11409116/40839479-7059145e-65ab-11e8-9b5b-177770a9434f.gif)

[example](http://jsbin.com/yarucanozo/1/edit)
 
### Expected behavior
The library had to generate an image

### Actual behavior (stack traces, console logs etc)
Generated image is empty

### Browsers

Chromium v. 66
Firefox v. 60

**Comment by simoneb** [comment]
Yep indeed it doesn't work. My workaround, when using Gatsby and the gatsby-image module, was to use a custom fragment which excludes the srcSet attribute.

**Comment by realamirhe** [comment]
any update on this issue?
